src/v/kafka/server/snc_quota_manager.cc

```diff
@@ -362,11 +362,11 @@ void snc_quota_manager::arm_balancer_timer() {
         // and on the first iteration
         if (
           _balancer_timer_last_ran.time_since_epoch()
-          != ss::lowres_clock::duration::zero()) {
+          != ss::lowres_clock::duration{}) {
```

Comment:
nit: this inequality parses unnaturally to me. i'd suggest considering using something like a std::optional to represent these cases explicitly just for readability.